dotnet/src/AutoGen.Anthropic/AnthropicClient.cs

```diff
@@ -61,33 +61,75 @@ public AnthropicClient(HttpClient httpClient, string baseUrl, string apiKey)
         using var reader = new StreamReader(await httpResponseMessage.Content.ReadAsStreamAsync());
 
         var currentEvent = new SseEvent();
+
         while (await reader.ReadLineAsync() is { } line)
         {
             if (!string.IsNullOrEmpty(line))
             {
-                currentEvent.Data = line.Substring("data:".Length).Trim();
+                if (line.StartsWith("event:"))
```

Comment:
The data payload of `Server side event` should always start with `data:`. Is that something that is anthropic specifc?

Reply:
Yes this is anthropic specific. This is an example response from docs

```
event: message_start
data: {"type": "message_start", "message": {"id": "msg_1nZdL29xx5MUA1yADyHTEsnR8uuvGzszyY", "type": "message", "role": "assistant", "content": [], "model": "claude-3-5-sonnet-20240620", "stop_reason": null, "stop_sequence": null, "usage": {"input_tokens": 25, "output_tokens": 1}}}

event: content_block_start
data: {"type": "content_block_start", "index": 0, "content_block": {"type": "text", "text": ""}}

event: ping
data: {"type": "ping"}

event: content_block_delta
data: {"type": "content_block_delta", "index": 0, "delta": {"type": "text_delta", "text": "Hello"}}

event: content_block_delta
data: {"type": "content_block_delta", "index": 0, "delta": {"type": "text_delta", "text": "!"}}

event: content_block_stop
data: {"type": "content_block_stop", "index": 0}

event: message_delta
data: {"type": "message_delta", "delta": {"stop_reason": "end_turn", "stop_sequence":null}, "usage": {"output_tokens": 15}}

event: message_stop
data: {"type": "message_stop"}
```